Compiling test-funcid-1-result-type-bounds-exception.pl0
Parsing complete
Static semantic analysis complete
Code generation complete
No errors detected.
Running ...
1
0
1
1
3

Runtime error: bounds check failed at 1106: 3 not in 0..2
PC=1107 in PROC   rangex : PROCEDURE) : int[0..2] level 1 start 1081 : 1114 FP=5 SL=0 DL=0 RA=1174
PC=1174 in PROC   <main> : PROCEDURE) : void level 0 start 1114 : 1176 FP=0 SL=0 DL=0 RA=0
  x(3) = 3
End of traceBack

Terminated
